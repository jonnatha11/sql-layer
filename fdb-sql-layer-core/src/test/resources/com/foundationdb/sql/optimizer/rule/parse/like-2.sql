SELECT * FROM parent WHERE name LIKE '_%%' ESCAPE '%'
