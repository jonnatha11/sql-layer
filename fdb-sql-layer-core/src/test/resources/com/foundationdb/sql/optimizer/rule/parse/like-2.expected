SelectQuery@2f566b7d
  ResultSet@52aaf3d2[id, name, state]
    Project@556d8a64[parent.id, parent.name, parent.state]
      Select@a98d70d[like(parent.name,'_%%','%')]
        TableSource@384e9bea(parent)
